styles/components/VersionCheck.module.less

```diff
@@ -0,0 +1,83 @@
+@import '../core/variables.less';
+
+.purple-centered {
+  color: @purple-dark;
+  text-align: center;
+}
+
+.mostly-opaque-background {
```

Comment:
This name isn't camel-cased/consistent with the others. (It also appears to not be in effect in the error states, so perhaps this is the cause?)